operator/v1/types_console.go

```diff
@@ -99,8 +99,60 @@ type StatuspageProvider struct {
 	PageID string `json:"pageID"`
 }
 
+// CapabilityName defines name of UI capability in the console UI.
+type CapabilityName string
```

Comment:
Given this is the operator package, maybe we should make this `ConsoleCapabilityName`

Reply:
I like that 👍